frontend/device/android/device.lua

```diff
@@ -347,12 +347,27 @@ end
 -- to-do: implement fullscreen toggle in API19+
```

Comment:
you can get rid of the entire function and comment here and just use

```lua
local api = android.app.activity.sdkVersion
```
in `Device:toggleFullscreen` directly.